_variables.tf

```diff
@@ -36,6 +36,13 @@ variable "multi_nat" {
   description = "Number of NAT Instances, 'true' will yield one per AZ while 'false' creates one NAT"
 }
 
+variable "multi_az_private_rtb" {
+  type        = bool
+  default     = false
+  description = "Create multiple private route tables, one per AZ"
```

Comment:
After looking at this change, i'm not sure how this option is different from multi_nat? Multi_nat already creates multiple private route tables, one per AZ and associate them with the NAT gateway.

Reply:
Hi @adenot,

The key difference is that the multi-nat setup creates one NATGateway per Availability Zone (AZ), which increases costs. Roughly an additional $33 per NATGateway per month, even if no data is processed through them. While this cost increase might be relatively minor, it still adds up.

By setting the variable multi_az_private_rtb, users get multi-AZ route tables to be used by a custom NATInstance solution, with a single NATGateway that should be used to failover in case one of the instances fails.